recipes/treelite/bld.bat

```diff
@@ -0,0 +1,12 @@
+mkdir build
+cd build
+cmake .. -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% -DENABLE_PROTOBUF=ON -GNinja
```

Comment:
```suggestion
cmake .. -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% -DCMAKE_BUILD_TYPE:STRING=Release -DENABLE_PROTOBUF=ON -GNinja
```

Reply:
That didn't solve the linkage error. Let's remove Windows support for now.